### ISSUE TITLE
USER STORY: Model optimization V9

### ISSUE THREAD

**Issue by Huddy2022** [Huddy2022/milestone-project-mildew-detection-in-cherry-leaves#28]
As a **data scientist** I want to **train the CNN model using the pre-processed dataset with re-sizes images 100 x 100 and optimize the hyperparameters reverting back to V2 but tweaking activation of sigmoid to softmax instead and changing loss - binary_crosstentropy to sparse_categorical_crossentropy instead to achieve the best performance,** so that **the model can accurately classify cherry leaves as healthy or containing powdery mildew.**
